### ISSUE TITLE
Build fails on python and scala plugins checksum

### ISSUE THREAD

**Issue by or-shachar** [bazelbuild/intellij#2343]
lastest master build fails on

`ERROR: /home/builduser/workspace/intellij-plugins-ci/intellij-check-and-notify-plugins-state/intellij/third_party/python/BUILD:7:13: //third_party/python:python_internal depends on @python_2020_1//:python in repository @python_2020_1 which failed to fetch. no such package '@python_2020_1//': java.io.IOException: Error downloading [https://plugins.jetbrains.com/files/7322/88054/python-ce-201.7846.93.zip] to /home/builduser/.cache/bazel/_bazel_builduser/1c03a469a8e977776a02273bfa495afd/external/python_2020_1/python-ce-201.7846.93.zip: Checksum was 648438d26e85072f90a62f9f5b9f9b983a49f3cb752e87d01f915cb6a03644b9 but wanted a3d7217794619dfa0432d922218dd875f49658939c6e5a54c01f99b333d35e12`

and

`Error in download_and_extract: java.io.IOException: Error downloading [https://plugins.jetbrains.com/files/1347/76628/scala-intellij-bin-2020.1.7.zip] to /home/builduser/.cache/bazel/_bazel_builduser/1c03a469a8e977776a02273bfa495afd/external/scala_2020_1/scala-intellij-bin-2020.1.7.zip: Checksum was 626b9c9bc2f90d64498524bd7138558edacf50bbc72a02f882401118a1fc1403 but wanted bbc019e7cde3baf21b11f0abde21f42d4606ca5ca9100b3bdace3292f70cceef`

I've checked the real checksum and it's `648438d26e85072f90a62f9f5b9f9b983a49f3cb752e87d01f915cb6a03644b9`. I guess CI here doesn't fail because it uses some kind of repository cache and takes the entry for `a3d7217794619dfa0432d922218dd875f49658939c6e5a54c01f99b333d35e12`

**Comment by cgruber** [comment]
latest release tag also fails with the same.

**Comment by cpsauer** [comment]
Ran into this, too, and was able to quickly apply your PR as a fix to [my repo of workarounds](https://github.com/hedronvision/bazelbuild-intellij). Thank you!

**Comment by or-shachar** [comment]
The problem I guess is with Jetbrrains... 
Why do they change the content of URL `https://plugins.jetbrains.com/files/7322/88054/python-ce-201.7846.93.zip` after it was posted? 

I wonder if we can open an issue there... I'd expect new URL for different content.

**Comment by or-shachar** [comment]
Opened this - https://intellij-support.jetbrains.com/hc/en-us/community/posts/360010424020-Consistent-files-in-plugin-repository

**Comment by YannCebron** [comment]
This happened to due signing all plugins https://blog.jetbrains.com/platform/2020/09/plugin-signing-in-marketplace/

https://twitter.com/JBPlatform/status/1356164386944135172?s=20

There are no plans/reasons for this occur again in the nearest future. 

New plugin are signed during the upload process, so they will differ from the original uploaded file.

**Comment by alice-ks** [comment]
Yann, thanks for the clarification. We had been wondering about the changed checksums and had hoped that adjusting them wouldn't be necessary again in the future.

Or, thanks for raising this issue and providing a fix. The CI just runs on newer IntelliJ versions and I guess that's why we haven't noticed it. In the last weeks, we fixed the checksums for the 2020.2 and 2020.3 plugins. I imported your PR to our internal source of truth (re-exported as #2371) so that the build will be also fixed for 2020.1.
